variants/AFROFLIGHT_F103CB/variant.h

```diff
@@ -0,0 +1,184 @@
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *******************************************************************************
+ */
+#ifndef _VARIANT_ARDUINO_STM32_
+#define _VARIANT_ARDUINO_STM32_
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        Pins
+ *----------------------------------------------------------------------------*/
+
+// Bluepill USB connector on the top, MCU side - Blackpill USB connector on bottom, MCU Side  (pins are reversed vertically for Arduino Ananlog pin correct sequence.
+// Left Side
+#define PB9  0
+#define PB8  1
+#define PB7  2
+#define PB6  3
+#define PB5  4
+#define PB4  5
+#define PB3  6
+#define PA15 7
+#define PA12 8  // USB DP
+#define PA11 9  // USB DM
+#define PA10 10
+#define PA9  11
+#define PA8  12
+#define PB15 13
+#define PB14 14
+#define PB13 15
+#define PB12 16 // LED Blackpill
+// Right side
+#define PC13 17 // LED Bluepill
+#define PC14 18
+#define PC15 19
+#define PA0  20 // A0
+#define PA1  21 // A1
+#define PA2  22 // A2
+#define PA3  23 // A3
+#define PA4  24 // A4 Board voltage ADC
+#define PA5  25 // A5
+#define PA6  26 // A6
+#define PA7  27 // A7
+#define PB0  28 // A8
+#define PB1  29 // A9
+#define PB10 30
+#define PB11 31
+// Other
+#define PB2  32 // BOOT1
+#define PA13 33 // SWDI0
+#define PA14 34 // SWCLK
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        35
+// This must be a literal with a value less than or equal to to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       10
+#define NUM_ANALOG_FIRST        20
+
+// On-board LED pin number
+#ifdef ARDUINO_BLUEPILL_F103C8
+#define LED_BUILTIN             PC13
+#else
+#define LED_BUILTIN             PB4
+#endif
+#define LED_GREEN               PB3
+#define LED_RED             LED_BUILTIN
+
+
+// SPI Definitions
+#define PIN_SPI_SS              PA4
+#define PIN_SPI_MOSI            PA7
+#define PIN_SPI_MISO            PA6
+#define PIN_SPI_SCK             PA5
+
+// I2C Definitions
+//#define PIN_WIRE_SDA            PB7
+//#define PIN_WIRE_SCL            PB6
+//On afroflight Rev 5 it's on i2c port 2 that MPU6050 is connected too
+#define PIN_WIRE_SDA      PB11
+#define PIN_WIRE_SCL      PB10
+
+
+// Timer Definitions
+// Do not use timer used by PWM pins when possible. See PinMap_PWM.
+#define TIMER_TONE              TIM3
+
+// Do not use basic timer: OC is required
+#define TIMER_SERVO             TIM2  //TODO: advanced-control timers don't work
```

Comment:
TIMER_Servo is optional so if the flight controller does not require it, you can remove it.